apps/docs/src/pages/en/guides/terraform.mdx

```diff
@@ -0,0 +1,60 @@
+---
+title: Terraform
+metaTitle: "Streamdal Terraform Provider"
+description: Information on Streamdal Terraform Provider
+layout: ../../../layouts/MainLayout.astro
+---
+
+Alternatively to defining pipelines via the UI, and to enhance pipeline 
+portability, we offer a [Terraform provider](https://registry.terraform.io/providers/streamdal/streamdal/latest). 
+This allows you to interact with the Streamdal Server API
```

Comment:
This allows you to define pipelines, steps, audiences and assignments programmatically.

Reply:
Updated